app/helpers/cases_helper.rb

```diff
@@ -285,7 +285,12 @@ def manager_updating_close_details_on_old_case?(user, kase)
     user.manager? && kase.closed? && !kase.allow_event?(user, :update_closure)
   end
 
+  # Note use of Case::FOI::Standard for Timeliness/ComplianceReview FOI cases
   def case_create_action(kase)
-    self.send("#{kase.model_name.singular}_index_path")
+    if kase.foi?
+      self.send("case_foi_standard_index_path")
+    else
+      self.send("#{kase.model_name.singular}_index_path")
+    end
```

Comment:
What do you think about getting rid of this conditional checking the type of the object by moving this logic into two methods, one on `Case::Base` and one on `Case::FOI::Standard` so you can just say `kase.case_create_action` ?

or maybe I am missing something - I'm not clear why we're doing this in the first place, TBH, so perhaps there's a good reason why it's this way

Reply:
Yes as you've noticed this is a total kludge which I have had to settle for this particular work around. It's a workaround because of the way `TimelinessReview` and `ComplianceReview` are not treated as fully bonafide resources still - i.e. they don't have their on `CRUD` routes - hence the check to process them using the `FOI::Standard -> create` route. It's also related to why the FOI Type selection radio buttons are not automatically selected when there are error messages displayed on FOI Case Creation.